### Graduate Students

### Getting Started in the Bik Lab

First, make sure you are familiar with our research - make sure you read the "Research" page of the Lab Website, to familariize yourself with research themes and our ongoing projects: [https://biklab.github.io/projects/](https://biklab.github.io/projects/)

Please also make sure you have completed all Lab Safety trainings, if you have not already done so. See the list of required modules on the main lab onboarding page: 

#### Bioinformatics training

Our lab aims to equip students with an interdisciplinary skill set, particularly emphasizing computational skill dvelopment alongside traditional molecular lab tasks (e.g. DNA extraction, PCR, Librarry prep for High-throughput Sequencing) and nematode taxonomy (sample processing, microscopy, slide mounting, species identification, use and interpretation of pictorial and dichotomous keys). Command line training is difficult and requires perserverence. 

To get oriented in the Bik Lab, we ask that you first complete these tutorials to gain some foundational bioinformatics skills:
	
* [Introduction to the Command Line (Unix Shell) - Software Carpentry](http://swcarpentry.github.io/shell-novice/)
* [R for Reproducible Scientific Analysis - Software Carpentry](http://swcarpentry.github.io/r-novice-gapminder/)
* [Programming with R - Software Carpentry](http://swcarpentry.github.io/r-novice-inflammation/)
* [Programming with Python - Software Carpentry](http://swcarpentry.github.io/python-novice-inflammation/)
* [QIIME2 introductory tutorial](https://docs.qiime2.org/2017.12/tutorials/moving-pictures/) - "Moving pictures of the human microbiome" (16S rRNA amplicon dataset that takes you through data processing, OTU picking, and ecological diversity analyses)
* (Remember: Googling your error messages is the secret to bioinformatics!

#### Key Literature to Read

Students are also expected to read the scientific literature, and think conceptually about _*why*_ we are carrying out different lab protocols and data analysis. **If you are waiting for assistance or find yourself with extra time, we expect you to always be reading the literaure!** 

Here are some review papers and references related to the work we do in the lab - it's a good idea to read these first:

* Creer et al. (2016) [The ecologists field guide to sequenced-based identification of biodiversity](https://biklab.github.io/assets/pdfs/Creer-etal-2016-MEE-EcolFieldGuide.pdf) _Methods in Ecology and Evolution_
* Bik et al. (2012) [Sequencing our way to understanding global eukaryotic biodiversity](https://biklab.github.io/assets/pdfs/Bik-2012-TREE-EukReview.pdf), _Trends in Ecology and Evolution_

The above review papers are just a starting point - for every reserach area mentioned in these papers you should be looking at the cited references (big list of journal articles referred to by author/number and listed at the end of the paper). Review papers only summarize what others have found; the real work is published in the primary literature (original hypothesis-driven reserach studies).

[Use Google Scholar](https://scholar.google.com/) to look up papers (paste in the title) and search for other papers within the same reserach field (use a combination of relevant keywords like: metagenomics, DNA sequencing, nematodes, marine sediments, etc.). 

Finally, let us know what specific area you're interested in exploring - for example, looking at signatures of evolution in nematode genomes, or analysing rRNA metabarcoding datasets to identify global biogeographic patterns in a specific eukaryotic taxon. These are just two ideas, but please chat with Holly and other lab members to identify specific areas suited to **your own areas of interest**. 
	
	
	
